articles/client-platforms/angular2/00-intro.md

```diff
@@ -40,45 +45,32 @@ Throughout the steps, the seed and samples will have the following directory str
 ```
 
 ## 4. Dependencies
-To integrate your Angular2 application with Auth0 you need to add the following dependencies:
 
-- [Lock Widget](https://github.com/auth0/lock): is the default authentication widget provided by Auth0.
+To integrate your Angular2 application with Auth0, you will need to add the following two dependencies:
 
-    From [npm](https://npmjs.org):
+- [Lock Widget](https://github.com/auth0/lock) is the default authentication widget provided by Auth0.
 
-    ```sh
-    npm install --save auth0-lock
-    ```
+  From [npm](https://npmjs.org):
 
-    From [bower](http://bower.io):
+  `npm install --save auth0-lock`
 
-    ```sh
-    bower install auth0-lock
-    ```
+  From [bower](http://bower.io):
 
-    Or our CDN:
+  `bower install auth0-lock`
 
-    ```html
-    <script src="http://cdn.auth0.com/js/lock/10.0.0/lock.min.js"></script>
-    ```
+  Or the Auth0 CDN:
 
-- [angular2-jwt](https://github.com/auth0/angular2-jwt): is a helper library for working with [JWTs](http://jwt.io/introduction) in your Angular 2 applications.
+  `<script src="http://cdn.auth0.com/js/lock/10.0.0/lock.min.js"></script>`
 
-    From [npm](https://npmjs.org):
+- [angular2-jwt](https://github.com/auth0/angular2-jwt) is a helper library for working with [JWTs](http://jwt.io/introduction) in your Angular 2 applications.
 
-    ```sh
-    npm install --save angular2-jwt
-    ```
+  From [npm](https://npmjs.org):
 
-## 5. Viewport
+  `npm install --save angular2-jwt`
 
-If you are targeting mobile audiences, it's recommended that you add in `<head>` element:
+## 5. Viewport
 
-```html
-<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=0"/>
-```
+If you are targeting mobile audiences, it's recommended that you add the following in the `<head>` element:
 
-## Done!
+`<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=0"/>`
 
-That's all you need to start working with Auth0!  
-Please continue with [login](/quickstart/spa/angular2/01-login) tutorial to know how to implement basic login.
```

Comment:
Links in this form do not work, they always get redirected to the intro page, as do all refreshes.